tests/misc_env/push_warp_workload.py

```diff
@@ -126,21 +132,28 @@ def run(ceph_cluster: Ceph, **kwargs) -> int:
         0 on Success and 1 on Failure.
     """
     LOG.info("Push workload to warp.")
-    client = ceph_cluster.get_nodes(role="client")[0]
+    client = ceph_cluster.get_nodes(role="installer")[0]
     warp_server = get_nodes_by_ids(ceph_cluster, kwargs["config"]["warp_server"])
     warp_clients = get_nodes_by_ids(ceph_cluster, kwargs["config"]["warp_client"])
     ha_clients = get_nodes_by_ids(ceph_cluster, kwargs["config"]["haproxy_client"])
     run_duration = kwargs["config"].get("duration", "1m")
+    warp_operation = kwargs["config"].get("operation", "fill")
```

Comment:
@udaysk23 , because of this the command will become "warp fill ..."
previously it was "warp put.."
is this intentional?

Reply:
@hmaheswa fixed